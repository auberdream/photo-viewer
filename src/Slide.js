import React, { Component } from 'react';

class Slide extends Component {
  render() {
    return (
      <div className="slide">
      </div>
    )
  }
}

export default Slide;
