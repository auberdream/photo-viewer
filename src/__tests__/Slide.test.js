import React from 'react';
import { shallow } from 'enzyme';
import Slide from '../Slide';

it('renders without crashing', () => {
  shallow(<Slide />);
})
