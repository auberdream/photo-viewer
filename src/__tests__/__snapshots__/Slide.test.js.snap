// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide renders correctly 1`] = `
<div
  className="slide"
>
  <img
    src="https://www.imasecuresite.com"
  />
  <div
    className="slide-information"
  >
    <div
      className="information-container"
    >
      <p
        className="photo-date"
      >
        2018-04-17
      </p>
      <p
        className="photographer-name"
      >
        Alan Berryhill
      </p>
      <p
        className="photo-tags"
      >
        Tags: person, outdoors
      </p>
    </div>
  </div>
</div>
`;
